message-index/messages/GHC-77037/example1/before/Warning.hs

```diff
@@ -0,0 +1,6 @@
+{-# OPTIONS_GHC -Wmissing-import-lists #-}
+
+module Warning where
+
+import Data.Maybe
```

Comment:
Could you either split ti to two examples or order of imports in before/after in the same way?